### DISCUSSION TITLE
Meeting Map & List Icons - Can this be text?

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1375] [Q&A]
I've had feedback from our audience asking where the meeting map is? 

Is it possible for the meeting Map & List icons to be changed to text? If so, how do I make this edit?

**Comment by joshreisner** [comment]
currently there's no way to customize this - however we're open to changing this for everyone to use text instead of icons - we're discussing it internally and will get back to you soon

**Reply by joshreisner** [reply]
@nataliefaith2 we've replaced the icons with text now
